src/index.html

```diff
@@ -4,10 +4,265 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <article class="header__container">
+        <article class="header__menu">
+          <!--BOSE-logo-->
+          <section class="header__logo">
+            <a href="#home">
+              <img
+                class="header__logo-img"
+                src="/images/BOSE.svg"
+                alt="bose-logo"
+              >
+            </a>
+          </section>
+          <section class="header__buttons">
+            <!--button-for-phone-number-->
+            <a href="tel:+1 234 5555-55-55" class="header__phone">
+              <img
+                class="header__phone-img"
+                src="/images/phone.svg"
+                alt="phone-number"
+              >
+            </a>
+            <!--button-for-burger-menu-->
+            <a href="#menu" class="header__toggle">
+              <span class="header__top-line"></span>
+              <span class="header__center-line"></span>
+              <span class="header__bottom-line"></span>
+            </a>
+          </section>
+        </article>
+        <!--opened-menu-->
+        <nav class="header__nav nav nav--closed">
```

Comment:
<img width="183" alt="Screenshot 2023-02-14 at 3 58 53 PM" src="https://user-images.githubusercontent.com/35467335/218760261-712cb113-7ef1-44bd-baa5-975d53516d7e.png">

You have different count of lines in nav icon for desktop and mobile.

Reply:
Yeah, I know that. Just at 320px there should be three lines on the menu icon, and already from 788px there should be four lines

Reply:
Yes, this was my mistake, sorry